user/models.py

```diff
@@ -1,3 +1,7 @@
 from django.db import models
 
-# Create your models here.
+class User(models.Model):
```

Comment:
Why have you avoided subclassing from one of the provided abstract user models, as described in the docs?